main.go

```diff
@@ -1,13 +1,12 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	fight "github.com/Nicolhime/rock-paper-scissors/fight"
```

Comment:
do not need to have the flight before the import. Basically read the import as the last thing after the slash. So it would be fight in this example


```suggestion
	"github.com/Nicolhime/rock-paper-scissors/fight"
```